README.md

```diff
@@ -1,133 +1,31 @@
-# Julee Example - Docker Setup Guide
+# Julee
```

Comment:
What does it stand for, actually? (Not that it needs to go here, it's just where I'd expect to find out - if anything)

Reply:
Hah - OK.

<img width="799" height="99" alt="image" src="https://github.com/user-attachments/assets/f6c292ca-86f8-4b27-8a7c-b1f815c801a7" />

Well *that* sounds easy to implement!